### PR TITLE
[FEATURE] Redesign Options Page

### PR DESCRIPTION
**Pull request by phacks**
Redesigns the options page for the extension.

Also simplifies the process for setting error messages - instead of `authenticationCheck: { success: false, message: "Failed to login" }`  we're now just using `authenticationErrorMessage: "Failed to login"` since we can tell the success state from the presence of a message

|                      | Before                                                                                                                                             | After                                                                                                                                              |
|----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
| Completed flow       | CleanShot 2023-06-01 at 14 31 08@2x | CleanShot 2023-06-01 at 14 26 28@2x |
| Save screen          | CleanShot 2023-06-01 at 14 31 21@2x | CleanShot 2023-06-01 at 14 26 49@2x |
| Default screen       | CleanShot 2023-06-01 at 14 28 36@2x | CleanShot 2023-06-01 at 14 30 00@2x |
| Authentication error | CleanShot 2023-06-01 at 14 30 32@2x | CleanShot 2023-06-01 at 14 27 20@2x |
| API Key warning      | CleanShot 2023-06-01 at 14 34 56@2x | CleanShot 2023-06-01 at 14 35 30@2x |

(After taking all of the above screenshots I decided to change the workspace selector to divide workspaces into two lists only if there are more than 5. Therefore:

CleanShot 2023-06-01 at 14 39 34@2x
CleanShot 2023-06-01 at 14 40 41@2x

I just couldn't be bothered to go take all the screenshots again)

### CODE REVIEW

File: src/options/options.html [hunk 1]
```diff
@@ -1,70 +1,128 @@
...
-          Successfully signed in! If you wish you sign out, visit your <a
-            href="https://app.orbit.love/user/settings/authorized_applications">User Settings on
-            Orbit</a> & revoke access for the "Orbit Browser Extension"
-          application.
-        </p>
+  <body>
+    <div class="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8">
+      <div class="mx-auto max-w-3xl sm:mt-10 lg:mt-20">
+        <div x-data="orbit">
+          <section>
```
Comment (phacks) [inline src/options/options.html:14]: Here I’m using `<section>`s for the major “blocks” of the extension:

- The presentation text + login part
- Selecting a workspace

File: src/options/options.html [hunk 2]
```diff
@@ -1,70 +1,128 @@
...
+            x-show="['success', 'warning'].includes(authenticationCheckStatus.status) && workspaces && workspaces.length> 0"
+          >
+            <div class="py-5">
+              <label class="text-base font-semibold text-gray-900"
+                >Select a workspace</label
+              >
+              <p class="text-sm text-gray-500">
+                This workspace will be used to retrieve information about community members, and create new ones.
+              </p>
+              <fieldset class="mt-4">
```
Comment (phacks) [inline src/options/options.html:77]: This is coming in from Tailwind UI: https://tailwindui.com/components/application-ui/forms/radio-groups

**Comment by puemos** [comment]
Idea: What if we change the radio buttons to a select?

**Comment by delete-44** [comment]
> Idea: What if we change the radio buttons to a select?

@puemos 

Honestly I'm not too sure what the best practice is for one or the other.

General consensus seems to be "if there's 6-7 or more items, or the items are very similar, use a select". 
https://joechiang.com/post/when-use-radio-buttons-select-dropdowns
https://ux.stackexchange.com/questions/456/when-should-i-use-a-select-box-instead-of-radio-buttons
https://blog.prototypr.io/7-rules-of-using-radio-buttons-vs-drop-down-menus-fddf50d312d1

I would argue that radio buttons suit this use case more, since the options are all completely different, there's no recommended default, & it's helpful to see them all at a glance.

Is there any information on the average number of workspaces a user is in?

**Review by phacks — COMMENTED** [review]
I can’t approve as I originally opened the draft, but it’s all good on my end! 👏

**Pull request merged**